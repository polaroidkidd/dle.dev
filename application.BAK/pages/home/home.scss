@import "../../styles/main";

.home {
  display: grid;
  grid-template-columns: 1fr;
  justify-items: center;
  margin: 2rem 1rem;
}