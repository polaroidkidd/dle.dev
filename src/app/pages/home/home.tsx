import React from "react";
import image from "../../assets/headshot/daniel_einars-400x500.jpg";
import { HeadingPrimary, HeadingSecondary, Navigation, ProfilePicture, WelcomeMessage } from "../../components";
import "./home.scss";

const Home: React.FC = () => {
  return (
    <section className=".home">
      <ProfilePicture
        imageSrc={image}
        isLarge={true}/>
      <HeadingPrimary
        text="dle"/>
      <Navigation/>
      <WelcomeMessage heading="Welcome to my corner of the internet."/>
      <HeadingSecondary text="This is currently a work in progress. Not much here to see yet."/>


    </section>
  );
};
export { Home };