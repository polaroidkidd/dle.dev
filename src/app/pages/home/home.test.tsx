import React from "react";
import renderer from "react-test-renderer";
import { Home } from "./home";

test("Expect to render complete Home", () => {
  const tree = renderer.create(<Home/>).toJSON();
  expect(tree).toMatchSnapshot();
});