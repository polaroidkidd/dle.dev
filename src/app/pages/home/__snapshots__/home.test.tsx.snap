// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to render complete Home 1`] = `
<section
  className=".home"
>
  <section>
    <h2
      className="heading-secondary welcome-message"
    >
      Hi! I'm Daniel Einars.
    </h2>
    <h3
      className="heading-tertiary u-center-text__horizontal u-center-text__padding"
    >
      I build websites & servers.
    </h3>
    <h4
      className="heading-quaternary u-center-text__horizontal"
    >
      Check out my 
      <a
        aria-current={null}
        href="/blog"
        onClick={[Function]}
      >
        first blog entry
      </a>
       on using React with Typescript!
    </h4>
  </section>
</section>
`;
